tests/conftest.py

```diff
@@ -28,39 +28,45 @@
 import shutil
 
 import pytest
-from reana_db.models import Base, User
+from flask import Flask
+from mock import Mock
 from sqlalchemy import create_engine
 from sqlalchemy.orm import scoped_session, sessionmaker
 from sqlalchemy_utils import create_database, database_exists, drop_database
 
+from reana_commons.api_client import BaseAPIClient
+from reana_db.database import Session
+from reana_db.models import Base, User
+from reana_server.config import COMPONENTS_DATA
 from reana_server.factory import create_app
 
 
 @pytest.fixture(scope='module')
-def tmp_shared_volume_path(tmpdir_factory):
-    """Fixture temporary file system database."""
-    temp_path = str(tmpdir_factory.mktemp('data').join('reana'))
-    shutil.copytree(os.path.join(os.path.dirname(__file__), "data"),
-                    temp_path)
-
-    yield temp_path
-    shutil.rmtree(temp_path)
-
-
-@pytest.fixture(scope='module')
-def base_app(tmp_shared_volume_path):
+def base_app():
     """Flask application fixture."""
     config_mapping = {
+        'AVAILABLE_WORKFLOW_ENGINES': 'serial',
         'SERVER_NAME': 'localhost:5000',
         'SECRET_KEY': 'SECRET_KEY',
         'TESTING': True,
-        'SHARED_VOLUME_PATH': tmp_shared_volume_path,
+        'SHARED_VOLUME_PATH': '/tmp/test',
```

Comment:
This is because you need https://github.com/reanahub/reana-pytest-commons/issues/4 before right?